integration_tests/features/steps/fal_steps.py

```diff
@@ -215,19 +233,18 @@ def _get_fal_results_file_name(context):
     return list(filter(lambda file: pattern.match(file), target_files))
 
 
-def _flatten_list(target_list):
+def _flatten(target_iterable):
     flat_list = []
-    for element in target_list:
-        if type(element) is list:
-            for item in element:
-                flat_list.append(item)
+    for element in target_iterable:
+        if isinstance(element, list):
+            flat_list.extend(element)
```

Comment:
The name change suggests that we could get non-lists, right? (e.g. `_flatten( [(1,2),(3,4)] )`) Not sure if this should give `[(1,2),(3,4)]` or `[1,2,3,4]`.

Reply:
> The name change suggests that we could get non-lists, right?

Yep! For example in `_get_models_from_result`, we pass it a `map()` object instead of a list.

> (e.g. _flatten( [(1,2),(3,4)] )) Not sure if this should give [(1,2),(3,4)] or [1,2,3,4].

I think what flatten is intended right now is just unpacking top-level lists in any iterable (a `map()` object, a list, etc.) to a list. Maybe we could rename it to something like `_unpack_top_level_lists(...)` but not sure. Or we could advance it with adding support to what you said but since this is a test utility, I don't think it would make that much difference. WDYT?

Reply:
i think its totally fine leave as is..